testing/integration_test.go

```diff
@@ -103,7 +111,13 @@ mkdir -p "$TESTDIR"
 cp -r "testing/busybench" "$TESTDIR"
 cd "$TESTDIR/busybench"
 
-retry npm install --nodedir="$NODEDIR" "$PROFILER" typescript gts >/dev/null
+retry npm install node-pre-gyp
+{{if eq .BinaryHost ""}}
+retry npm install --nodedir="$NODEDIR" --build-from-source=profiler "$PROFILER" typescript gts >/dev/null
+{{else}}
+retry npm install --nodedir="$NODEDIR" --fallback-to-build=false --profiler_binary_host_mirror={{.BinaryHost}} "$PROFILER" typescript gts >/dev/null
```

Comment:
Where is --profiler_binary_host_mirror flag name coming from? E.g. https://www.npmjs.com/package/node-pre-gyp has
```
npm install v8-profiler --profiler_binary_host_mirror=https://npm.taobao.org/mirrors/node-inspector/
```
as example, that uses the same flag name, is that a problem?

Reply:
I'm not totally sure how to answer this as I don't fully understand the implications. Is this a globally and externally visible name? For the overall profiler module we use @google-cloud/profiler - is this a different name?

Reply:
This name can't have dashes, @s, or slashes. It has to basically be a valid C variable name (and I think all lowercase).

It's probably best if it's unique-ish, so should switch to  google_cloud_profiler.

Reply:
This is done. Marking as resolved.